custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js

```diff
@@ -56,10 +56,14 @@ function AdditionalFilterController($scope, $location, $uibModal, storageService
     vm.selectedGender = $location.search()['gender'] !== void(0) ? $location.search()['gender'] : '';
     vm.selectedAge = $location.search()['age'] !== void(0) ? $location.search()['age'] : '';
     var filtersObjects = [];
+    // eg:vm.filters = ['gender', 'age']
+    // this array has filters that are not to be shown. so if 'gender' is not in array, it can be shown.
     if (vm.filters && vm.filters.indexOf('gender') === -1) {
+        vm.showGenderFilter = true;
```

Comment:
should these be initialized to `false`? (I realize `undefined` also works, but think it's clearer to be explicit)

Reply:
the other option is to have
```javascript
vm.showGenderFilter = function() {
  return vm.filters && vm.filters.indexOf('gender') === -1;
}
```
which was what I was thinking of originally

Reply:
(both of these are optional IMO but will let you decide if you want to do either one)

Reply:
I'm going to initialize variables to false. But will let other things stay as is.